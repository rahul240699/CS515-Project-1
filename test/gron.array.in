python3 prog/gron.py array.json