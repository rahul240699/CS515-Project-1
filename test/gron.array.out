json = [];
json.[0] = {};
json.[0].id = "0001";
json.[0].type = "donut";
json.[0].name = "Cake";
json.[0].ppu = 0.55;
json.[0].batters = {};
json.[0].batters.batter = [];
json.[0].batters.batter[0] = {};
json.[0].batters.batter[0].id = "1001";
json.[0].batters.batter[0].type = "Regular";
json.[0].batters.batter[1] = {};
json.[0].batters.batter[1].id = "1002";
json.[0].batters.batter[1].type = "Chocolate";
json.[0].batters.batter[2] = {};
json.[0].batters.batter[2].id = "1003";
json.[0].batters.batter[2].type = "Blueberry";
json.[0].batters.batter[3] = {};
json.[0].batters.batter[3].id = "1004";
json.[0].batters.batter[3].type = "Devil's Food";
json.[0].topping = [];
json.[0].topping[0] = {};
json.[0].topping[0].id = "5001";
json.[0].topping[0].type = "None";
json.[0].topping[1] = {};
json.[0].topping[1].id = "5002";
json.[0].topping[1].type = "Glazed";
json.[0].topping[2] = {};
json.[0].topping[2].id = "5005";
json.[0].topping[2].type = "Sugar";
json.[0].topping[3] = {};
json.[0].topping[3].id = "5007";
json.[0].topping[3].type = "Powdered Sugar";
json.[0].topping[4] = {};
json.[0].topping[4].id = "5006";
json.[0].topping[4].type = "Chocolate with Sprinkles";
json.[0].topping[5] = {};
json.[0].topping[5].id = "5003";
json.[0].topping[5].type = "Chocolate";
json.[0].topping[6] = {};
json.[0].topping[6].id = "5004";
json.[0].topping[6].type = "Maple";
json.[1] = {};
json.[1].id = "0002";
json.[1].type = "donut";
json.[1].name = "Raised";
json.[1].ppu = 0.55;
json.[1].batters = {};
json.[1].batters.batter = [];
json.[1].batters.batter[0] = {};
json.[1].batters.batter[0].id = "1001";
json.[1].batters.batter[0].type = "Regular";
json.[1].topping = [];
json.[1].topping[0] = {};
json.[1].topping[0].id = "5001";
json.[1].topping[0].type = "None";
json.[1].topping[1] = {};
json.[1].topping[1].id = "5002";
json.[1].topping[1].type = "Glazed";
json.[1].topping[2] = {};
json.[1].topping[2].id = "5005";
json.[1].topping[2].type = "Sugar";
json.[1].topping[3] = {};
json.[1].topping[3].id = "5003";
json.[1].topping[3].type = "Chocolate";
json.[1].topping[4] = {};
json.[1].topping[4].id = "5004";
json.[1].topping[4].type = "Maple";
json.[2] = {};
json.[2].id = "0003";
json.[2].type = "donut";
json.[2].name = "Old Fashioned";
json.[2].ppu = 0.55;
json.[2].batters = {};
json.[2].batters.batter = [];
json.[2].batters.batter[0] = {};
json.[2].batters.batter[0].id = "1001";
json.[2].batters.batter[0].type = "Regular";
json.[2].batters.batter[1] = {};
json.[2].batters.batter[1].id = "1002";
json.[2].batters.batter[1].type = "Chocolate";
json.[2].topping = [];
json.[2].topping[0] = {};
json.[2].topping[0].id = "5001";
json.[2].topping[0].type = "None";
json.[2].topping[1] = {};
json.[2].topping[1].id = "5002";
json.[2].topping[1].type = "Glazed";
json.[2].topping[2] = {};
json.[2].topping[2].id = "5003";
json.[2].topping[2].type = "Chocolate";
json.[2].topping[3] = {};
json.[2].topping[3].id = "5004";
json.[2].topping[3].type = "Maple";